gulpfile.js

```diff
@@ -416,11 +421,31 @@ export function buildDocs() {
 
 export async function buildDocsWatch() {
   await buildDocs();
-
   console.log("Listening for changes in documentation...");
   return gulp.watch(sourceFiles, buildDocs);
 }
 
+function combineForSandcastle() {
+  const outputDirectory = join("Build", "Sandcastle", "CesiumUnminified");
+  return build({
+    minify: false,
+    removePragmas: false,
+    outputDirectory: outputDirectory,
```

Comment:
Does the outputDirectory argument matter here? It's not passed to buildCesium from build.